sdk/python/tests/test_client.py

```diff
@@ -201,7 +202,7 @@ def test_version(self, mocked_client, mocker):
         [pytest.lazy_fixture("mock_client"), pytest.lazy_fixture("secure_mock_client")],
     )
     def test_get_online_features(self, mocked_client, mocker):
-        ROW_COUNT = 300
+        ROW_COUNT = 1
```

Comment:
Why was the count changed here?

Reply:
Corrected.

Reply:
Hmmm, it is still showing as `ROW_COUNT = 1` to me. Is that expected?

Reply:
:disappointed_relieved: Should be reverted.